modules/python/src2/hdr_parser.py

```diff
@@ -627,6 +627,8 @@ def parse_func_decl(self, decl_str, mat="Mat", docstring=""):
                                                              ("noArray", arg_type)]).strip()
                     if '/IO' in modlist and '/O' in modlist:
                         modlist.remove('/O')
+                    if arg_type == 'String' and arg_name == 'filename':
+                        modlist.append('/PATH')
```

Comment:
There are other cases:
- read/write methods with parameter `fileName` or type `char*`
-  `filepath` parameter name
- a lot of methods in DNN with several parameters with path semantic like https://docs.opencv.org/4.x/d6/d0f/group__dnn.html#ga29d0ea5e52b1d1a6c2681e3f7d68473a or https://docs.opencv.org/4.x/db/d30/classcv_1_1dnn_1_1Net.html#a327720af187378a58e0a48feb0b5ef89

Reply:
OK, then it makes sense to go with the idea from the original issue to be able to label parameters as paths (e.g.  `CV_WRAP_FILE_PATH`)

Reply:
Pushed